src/components/Upload.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Is there an example of how to connect i18 with redux state somewhere in the genric ico?